### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by nwtn**
* update README with Rails instructions
* update CONTRIBUTING with release steps
* update CHANGELOG for 1.1.0 release (to come once this merges)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 <!-- ## [Unreleased] -->
 
+## 1.1.0 - 2018-04-20
+
+Polaris tokens are now available as both a [JS package](https://www.npmjs.com/package/@shopify/polaris-tokens) and a [Ruby gem](https://rubygems.org/gems/polaris_tokens)! Check the [README](https://github.com/Shopify/polaris-tokens/blob/master/README.md) for installation and usage instructions for both.
```
Comment (kaelig) [inline CHANGELOG.md]: Suggestion: JS package -> npm package

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -174,3 +174,11 @@ yarn test -u
 ```
 
 [issue-template-color-updated]: https://github.com/Shopify/polaris-tokens/issues/new?template=update-colors.md&title=Update%20colors%20🎨
+
+## Releasing
+
+When releasing a new version, follow these steps:
```
Comment (kaelig) [inline CONTRIBUTING.md]: Mention "on `master`"?

File: README.md [hunk 3]
```diff
@@ -22,9 +22,11 @@ Design tokens originated at Salesforce, and the best way to describe them is to
 
 ## Installation
 
-Polaris design tokens are available on [npm](https://www.npmjs.com/) as the `@shopify/polaris-tokens` package.
+Polaris design tokens are available as both a JavaScript package (`@shopify/polaris-tokens`) on [npm](https://www.npmjs.com/), and as a Ruby gem (`polaris_tokens`) on [RubyGems](https://rubygems.org/).
 
-The recommended way to use and install design tokens may vary depending on your project, the most common are documented below.
+### JavaScript package installation
+
+The recommended way to use and install design tokens may vary depending on your project; the most common are documented below.
```
Comment (kaelig) [inline README.md]: Should this line be above `### JavaScript package installation`?

File: README.md [hunk 4]
```diff
@@ -22,9 +22,11 @@ Design tokens originated at Salesforce, and the best way to describe them is to
 
 ## Installation
 
-Polaris design tokens are available on [npm](https://www.npmjs.com/) as the `@shopify/polaris-tokens` package.
+Polaris design tokens are available as both a JavaScript package (`@shopify/polaris-tokens`) on [npm](https://www.npmjs.com/), and as a Ruby gem (`polaris_tokens`) on [RubyGems](https://rubygems.org/).
```
Comment (kaelig) [inline README.md]: Ditto - JavaScript package -> npm package?

**Comment by nwtn** [comment]
updated

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 <!-- ## [Unreleased] -->
 
+## 1.1.0 - 2018-04-20
+
```
Comment (kaelig) [inline CHANGELOG.md]: Don't forget to update the date here

**Pull request merged**